tools/kube-capture/cmd/kube-capture/main.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
naming bikeshed: I don't like `kube-capture` because `kube` is too narrow scoped (this can, or at least in the future, be beyond kube) and `capture` is overloaded (iptables) and not that clear. Alternatives:
* istioctl bug-report
* istioctl dump
* istioctl dump-info

Strongly discourage `istioctl debug` as it may conflict with the future `kubectl debug` in some way

Reply:
bug-report it is.